packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -384,22 +378,27 @@ class TimelineFrame {
   int get cpuStartTime => _cpuEventFlow.startTime;
   int get cpuEndTime => cpuStartTime + cpuDuration;
   int get cpuDuration => _cpuEventFlow.duration;
+  double get cpuDurationMs => cpuDuration / 1000;
 
   // Timing info for GPU portion of the frame.
   int get gpuStartTime => _gpuEventFlow.startTime;
   int get gpuEndTime => gpuStartTime + gpuDuration;
   int get gpuDuration => _gpuEventFlow.duration;
+  double get gpuDurationMs => gpuDuration / 1000;
+
+  bool get isCpuSlow => cpuDurationMs > 8.0;
```

Comment:
instead of hard coding 8.0 here, use half the target frame  rate
at some point we will want to let you indicate that your FPS goal is 120fps on some devices.

Reply:
done.